### PR TITLE
Merchant item create

### PR DESCRIPTION
**Pull request by kenzjoy**
- created new_spec and new view for merchant items
- added a button on merchant items index for merchant to add new item
- created a form on merchant items new for merchant to fill out fields with new item attributes
- new item automatically is filed under "disabled items" on the merchant item index once form is submitted

### CODE REVIEW

**Review by MadelineMauser — APPROVED** [review]
We may want to refactor later so that the create uses strong params, but otherwise it looks good!

File: app/views/items/new.html.erb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+
+
+<%= form_with url: "/merchants/#{@merchant.id}/items", method: :post, local: true do |form| %>
+  <%= form.label :name %>
+  <%= form.text_field :name %>
+  <%= form.label :description %>
+  <%= form.text_field :description %>
```
[the remark this answers is not in the document]
Comment (msakr21) [reply app/views/items/new.html.erb:7]: I think that's an excellent idea!

**Comment by msakr21** [comment]
I resolved the conflict that existed because of the change to the code that I introduced. I will go ahead and submit my approval. I think for the text_area instead of text_field modification that Madeline mentioned, we can create a projects card for it and do later on to proceed with merging this.

**Review by msakr21 — APPROVED** [review]
I added my thoughts/suggestions/feedback in comments!

**Pull request closed without merging**